assets/foundation-flex.scss

```diff
@@ -1,3 +1,40 @@
+$foundation-components: (
```

Comment:
Is it usefull to duplicate component enabling settings in `assets/*` and `scss/foundation.scss` ? We could provide a boolean setting like `$foundation-components-enable` which enable or disable all components by default.